src/features/content-explorer/content-explorer/ContentExplorerBreadcrumbs.js

```diff
@@ -28,23 +28,25 @@ const ContentExplorerBreadcrumbs = ({
         >
             <IconChevron direction="left" size="6px" color="#333" />
         </Button>
-        <Breadcrumb label={formatMessage(messages.breadcrumb)}>
+        <Breadcrumb label={formatMessage(messages.breadcrumb)} {...breadcrumbProps}>
             {foldersPath.map((folder, i) => (
-                <PlainButton
+                <div
```

Comment:
What's the reason for this change? Is it style related?

Reply:
Yes, PlainButton does not play nice with Breadcrumbs